filters/filter_24_QuickFixes/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 24,
+  "name": "AdGuard Quick Fixes filter",
+  "description": "Filter to quickly resolve content filtering issues on popular websites without updating the extension.",
+  "timeAdded": 1724747105555,
+  "homepage": "https://adguard.com/kb/general/ad-filtering/adguard-filters/",
+  "expires": "5 days",
+  "displayNumber": 1,
```

Comment:
there are two filters for the mv3 extension now: 
- base filter with `"displayNumber": 1`
- mobile ads filters with `"displayNumber": 2`

so if the displayNumber for the new filter is `1`, it will be displayed before the mobile ads filter. is it ok?

Reply:
If that's the case, I think it's fine. @Alex-302 is it necessary to lower it down?
```
AdGuard Base
AdGuard Quick Fixes
AdGuard Mobile Ads
```

Reply:
ok

Reply:
Decided to change the `displayNumbler` value.